tests/unit/lib/reporters/JsonCoverage.js

```diff
@@ -0,0 +1,64 @@
+define([
+	'intern!object',
+	'intern/chai!assert',
+	'dojo/node!istanbul/lib/collector',
+	'dojo/node!istanbul/lib/report/json',
+	'dojo/node!fs',
+	'./support/mocks',
+	'../../../../lib/reporters/JsonCoverage'
+], function (registerSuite, assert, Collector, Reporter, fs, mock, JsonCoverage) {
+	var sessionId = 'foo';
+
+	registerSuite({
+		name: 'intern/lib/reporters/JsonCoverage',
+
+		coverage: function () {
```

Comment:
This is testing literal implementation detail (does the reporter call `add` on its internal collector?) rather than that information passed to `coverage` ends up in the final output of the reporter, so isn’t a very good test.